kickstarter/android-oss
# NTV-274: User model

# 📲 What
- Migrated to kotlin
- Adopt @Parcelize 

# 🤔 Why
- Remove all references to AutoParcel library in Models, eventually we will be able to remove that dependency.

# 👀 See
- No user facing changes

|  |  |

# 📋 QA
- Log In.
- Create new user.
- Check any of your previously backed projects, go the comments section, make sure you can comment.
- Look into your messages
- Pledge to a project
- Navigate over a few projects
- Log in as creator an take a look into the dashboard and a project created

# Story 📖

[NTV-274](https://kickstarter.atlassian.net/browse/NTV-274)
